src/genie/libs/parser/iosxe/show_policy_map.py

```diff
@@ -2300,7 +2300,8 @@ class ShowPolicyMapSchema(MetaParser):
                     Any(): {
                         Optional('priority_level'): {
                             Any(): {
-                                'kbps': int}},
+                                'kbps': int,
```

Comment:
Kindly revert back the changes to the original.

Reply:
corrected and restored.